code/modules/mob/living/brain/posibrain.dm

```diff
@@ -6,37 +6,47 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	icon = 'icons/obj/assemblies.dmi'
 	icon_state = "posibrain"
 	w_class = WEIGHT_CLASS_NORMAL
-	var/next_ask
-	var/askDelay = 600 //one minute
+	var/ask_role = "" ///Can be set to tell ghosts what the brain will be used for
```

Comment:
i think you need to put autodoc comments before the var or else it gives the autodoc to the var after it

Reply:
a lot of vars so far are commented like this, if that's actually the case there's a lot of fixing to do

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variable

Reply:
Does that mean that there is indeed a lot of fixing to do?

Reply:
It pulls from the line before afaik, so yeah. Either way, put them above for consistency with the rest of the codebase. Note, this applies to the /// autodoc comments, not the // non autodoc comments.